python/graphstorm/run/launch.py

```diff
@@ -1015,8 +1015,11 @@ def get_argument_parser():
     )
     parser.add_argument(
         "--do-nid-remap",
-        action="store_true",
-        help="Do GraphStorm node ID to Raw input node ID remapping."
+        type=lambda x: (str(x).lower() in ['true', '1']),
```

Comment:
can we add this argument to the docs/source/configuration/configuration-run.rst file?